### ISSUE TITLE
Error: Could not parse JSON (parse error: after array element, I expect ',' or ']'

### ISSUE THREAD

**Issue by chapeupreto** [tobi-wan-kenobi/bumblebee-status#792]
Hello there!

I'm getting the following error message when I try to run `bumblebee-status` on an `Ubuntu 20.04.2 LTS` system:

```bash
Error: Could not parse JSON (parse error: after array element, I expect ',' or ']')
```

I've installed bumblebee-status via `pip`.

My `bar` section is kind of the same as the one mentioned on the README:

```
bar {
	font pango:Inconsolata 10
	position top
	tray_output none
	status_command bumblebee-status -m nic disk:root cpu \
		memory battery date time pasink pasource dnf \
		-p root.path=/ time.format="%H:%M CW %V" date.format="%a, %b %d %Y" \
		-t solarized-powerline
}
```

When I run bumblebee-status right on my shell, I get the following output:

```
{"version": 1, "click_events": true}
[
[{"full_text": " wifi wlp0s20f3 up 192.168.1.100 Fatal_Error ", "color": "#aabbcc", "name": "nic", "instance": "nic::0", "_raw": "wlp0s20f3 up 192.168.1.100 Fatal_Error", "_suffix": " ", "_prefix": " wifi ", "align": null}, {"full_text": " tun tun1 up 10.8.0.46 ", "color": "#aabbcc", "name": "nic", "instance": "nic::1", "_raw": "tun1 up 10.8.0.46", "_suffix": " ", "_prefix": " tun ", "align": null}, {"full_text": " hdd 24.71GiB/937.40GiB (02.64%) ", "color": "#aabbcc", "name": "root", "instance": "6bb324d8-e227-41c4-ba93-abce528e8fc8", "_raw": "24.71GiB/937.40GiB (02.64%)", "_suffix": " ", "_prefix": " hdd ", "align": null}],
[{"full_text": " wifi wlp0s20f3 up 192.168.1.100 Fatal_Error ", "color": "#aabbcc", "name": "nic", "instance": "nic::0", "_raw": "wlp0s20f3 up 192.168.1.100 Fatal_Error", "_suffix": " ", "_prefix": " wifi ", "align": null}, {"full_text": " tun tun1 up 10.8.0.46 ", "color": "#aabbcc", "name": "nic", "instance": "nic::1", "_raw": "tun1 up 10.8.0.46", "_suffix": " ", "_prefix": " tun ", "align": null}, {"full_text": " hdd 24.71GiB/937.40GiB (02.64%) ", "color": "#aabbcc", "name": "root", "instance": "6bb324d8-e227-41c4-ba93-abce528e8fc8", "_raw": "24.71GiB/937.40GiB (02.64%)", "_suffix": " ", "_prefix": " hdd ", "align": null}],
^CTraceback (most recent call last):
  File "/home/rodiney/.local/bin/bumblebee-status", line 146, in <module>
    main()
  File "/home/rodiney/.local/bin/bumblebee-status", line 138, in main
    output.wait(config.interval())
  File "/home/rodiney/.local/lib/python3.8/site-packages/bumblebee_status/core/output.py", line 257, in wait
    time.sleep(interval)

```

And this is the output of running `i3 --version`:

```bash
i3 version 4.17.1 (2019-08-30) © 2009 Michael Stapelberg and contributors
```

Any ideas on how to solve this issue?

Thanks!

**Comment by tobi-wan-kenobi** [comment]
Weird, the console output looks good.

Can you maybe narrow it down to a specific module that causes the issue? (ie temove half of the modules, see if the issue is gone, etc. until only one module is the culprit)

For example, you can remove dnf right away, as Ubuntu uses apt, not dnf/yum.

Sorry for the issue!

**Comment by tobi-wan-kenobi** [comment]
Oh, one alternative: Maybe you need to provide the full path to `bumblebee-status` in the status_command. I am not entirely sure which paths are included by default.

**Comment by chapeupreto** [comment]
Thank you very much for your reply.

I did what you suggested: I first removed `dnf` and now everything is working great. There are so many Linux distros out there that I had no idea what dnf is.

One more thing that I had to do (not quite related to this issue) is that I had to install `psutil` in order to see my CPU utilization:

```bash
pip install psutil
```

This step is not on the README, but I can see it's mentioned on the docs here: https://bumblebee-status.readthedocs.io/en/main/index.html

So, I'm going to close this issue.

Thanks again for your help and thanks a lot for this amazing project.
